tests/integration-tests/tests/common/utils.py

```diff
@@ -38,16 +40,30 @@
     "ubuntu1804": {"name": "Deep Learning Base AMI (Ubuntu 18.04)*", "owners": ["amazon"]},
 }
 
-AMI_TYPE_DICT = {"official": OS_TO_OFFICIAL_AMI_NAME_OWNER_MAP, "remarkable": OS_TO_REMARKABLE_AMI_NAME_OWNER_MAP}
+OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP = {
+    "alinux": {"name": "-amzn-hvm-x86_64-*", "owners": ["247102896272"]},
+    "alinux2": {"name": "-amzn2-hvm-*-*", "owners": ["247102896272"]},
+    "centos6": {"name": "-centos6-hvm-x86_64-*", "owners": ["247102896272"]},
+    "centos7": {"name": "-centos7-hvm-x86_64-*", "owners": ["247102896272"]},
+    "ubuntu1604": {"name": "-ubuntu-1604-lts-hvm-x86_64-*", "owners": ["247102896272"]},
+    "ubuntu1804": {"name": "-ubuntu-1804-lts-hvm-*-*", "owners": ["247102896272"]},
+}
+
+AMI_TYPE_DICT = {
+    "official": OS_TO_OFFICIAL_AMI_NAME_OWNER_MAP,
+    "remarkable": OS_TO_REMARKABLE_AMI_NAME_OWNER_MAP,
+    "pcluster": OS_TO_PCLUSTER_AMI_NAME_OWNER_MAP,
+}
 
 
 def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
     ec2_client = boto3.client("ec2", region_name=region)
+    if ami_type == "pcluster":
+        name = "aws-parallelcluster-" + str(get_installed_version()) + AMI_TYPE_DICT[ami_type][os]["name"]
+    else:
+        name = AMI_TYPE_DICT[ami_type][os]["name"]
     response = ec2_client.describe_images(
```

Comment:
describe_images can raise a ClientError, are we managing it elsewhere?

Reply:
handled